### ISSUE TITLE
Remove version number from pmo base URI

### ISSUE THREAD

**Issue by NancyL** [LD4P/PerformedMusicOntology#28]
We have decided that the base URI for PMO should not include the /2.0/ version indicator. We also understand that in order to do this, we will "unpublish" PMO 1.0.

Example: 
http://performedmusicontology.org/2.0/ontology/AllMusicId
will be
http://performedmusicontology.org/ontology/AllMusicId

**Comment by jermnelson** [comment]
@NancyL a question about the structure of the URIs in 2.0 version.

In the source RDF, the subject URI has the following pattern (taken from [Ensemble_size.ttl](https://github.com/LD4P/PerformedMusicOntology/blob/main/2.0/Ontology/Vocabularies/EnsembleSize/Ensemble_size.ttl)), 
`http://performedmusicontology.org/2.0/vocabularies/ensemble_size/`, should this be changed to `http://performedmusicontology.org/vocabularies/ensemble_size/` and not `http://performedmusicontology.org/ontology/vocabularies/ensemble_size/`?

**Comment by NancyL** [comment]
yes, all should be without the 2.0... I guess I didn't get to those
